src/components/ArticleList.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import { getArticles } from '../utils';
+
+import Header from './PageHeader';
+import ArticleHeader from './ArticleHeader';
+
+export default function ArticleList() {
+  const [articles, setArticles] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getArticles()
+      .then(articles => {
+        setArticles(articles);
+        setIsLoading(false);
+      })
+      .catch(error => {
+        console.log(error);
```

Comment:
Instead of logging the error, you could set a state for the error and display a message conditionally.